graphstorm-processing/docker/0.3.0/emr/Dockerfile.cpu

```diff
@@ -0,0 +1,76 @@
+# TODO: Pin image version
+FROM public.ecr.aws/amazoncorretto/amazoncorretto:17 as base
```

Comment:
A general comment, is the difference between emr and emr-serverless only about basic image?

Reply:
No the EMR on EC2 image requires extra dependencies to be installed as it's more "bare bones" than the EMR-S image.